### PR TITLE
Different client and configs for HBase 2.x

### PR DESCRIPTION
**Pull request by Arun-Krishna**

### CODE REVIEW

File: bigtable/bigtable.sh
Comment (medb) [inline bigtable/bigtable.sh:112]: We need to create it in temp directory;

```bash
local hbase_config
hbase_config=$(mktemp /tmp/hbase-site.xml-XXXX)
```

File: bigtable/bigtable.sh
Comment (medb) [inline bigtable/bigtable.sh:157]: Use `[[` instead of `[`

File: bigtable/bigtable.sh
Comment (medb) [inline bigtable/bigtable.sh:165]: We usually make conditions based on the Dataproc image version, it's available in `DATAPROC_VERSION` env variable.

File: bigtable/bigtable.sh
Comment (medb) [inline bigtable/bigtable.sh:178]: Move then on the same line as `if` for consistency:

```bash
if [[ "$hbase_version" = "2" ]]; then
```

**Review by medb — CHANGES_REQUESTED** [review]
We will need to wait on new image release before merging this PR to make sure that GitHub presubmit passes.

File: bigtable/bigtable.sh
Comment (Arun-Krishna) [reply bigtable/bigtable.sh]: Done.

File: bigtable/bigtable.sh
Comment (Arun-Krishna) [reply bigtable/bigtable.sh]: Done.

File: bigtable/bigtable.sh
Comment (Arun-Krishna) [reply bigtable/bigtable.sh]: Done.

**Review by animeshnandanwar — APPROVED** [review]
LGTM

**Pull request merged**
